NuGet/Xamarin.Microsoft.Band.nuspec

```diff
@@ -14,26 +14,9 @@
         <releaseNotes></releaseNotes>
         <copyright>Microsoft and Xamarin Inc.</copyright>
         <tags>Microsoft Band,Fitness,Health,Xamarin,Android,iOS,windows phone,wp8,monodroid,monoandroid,monotouch</tags>
-        <dependencies>
-            <group targetFramework="netcore451">
-                <dependency id="Xamarin.Microsoft.Band.Native" version="1.3.8" />
-            </group>
-            <group targetFramework="win">
-                <dependency id="Xamarin.Microsoft.Band.Native" version="1.3.8" />
-            </group>
-            <group targetFramework="wpa81">
-                <dependency id="Xamarin.Microsoft.Band.Native" version="1.3.8" />
-            </group>
-            <group targetFramework="uap10">
-                <dependency id="Xamarin.Microsoft.Band.Native" version="1.3.8" />
-            </group>
-            <group targetFramework="monoandroid">
-                <dependency id="Xamarin.Microsoft.Band.Native" version="1.3.8" />
-            </group>
-            <group targetFramework="xamariniOS">
-                <dependency id="Xamarin.Microsoft.Band.Native" version="1.3.8" />
-            </group>
-        </dependencies>
+    <dependencies>
+      <dependency id="Xamarin.Microsoft.Band.Native" version="1.3.8" />
+    </dependencies>
```

Comment:
Unless a real reason, this needs to be separate dependencies. NuGet v2 will try and install the native NuGet into a PCL project. This either works for some profiles, but it installs the win81+wpa81 dll, or throws an exception saying that the portable profile cannot be found.